src/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <title>Capstone JS</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="nav">
+      <div class="app-logo">
+        <span class="material-symbols-outlined icon-play">
+          play_circle
+        </span>
+        Movies
+      </div>
+      <div class="app-navigation">
+        <ul class="nav-list">
+          <li class="nav-list-item"><a href="">Movies</a></li>
+          <li class="nav-list-item"><a href="">Shows</a></li>
+          <li class="nav-list-item"><a href="">Shows 2</a></li>
+        </ul>
+      </div>
+      <div class="user-navigation">
+        <ul class="nav-list">
+          <li class="profile-container">
+            <div class="profile"> <i class="text-dark fa fa-user"></i></div>
+            <!-- <i class="profile-arrow-down fa fa-arrow-down"></i> -->
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main class="main">
+    <section class="item-counter">
+       <div></div>
+       <div class="counter">
+          <p>displayed movies</p>
+          <i class="icon fa fa-chevron-right"></i>
+          <p class="" id="numberOfItems">5</p>
+       </div>

```

Comment:
- [ ] Kindly positioned this element in accordance to the given template; more so, you need to the display the exact value  of the counter. ℹ️ 

## Your Design
<img width="917" alt="Untitled" src="https://github.com/rodrigopenayo98/capstone-javascript-rod-haben/assets/103386183/740e69c2-7062-4ef8-b343-c6f19ca0641d">

## Given template
<img width="427" alt="Untitled" src="https://github.com/rodrigopenayo98/capstone-javascript-rod-haben/assets/103386183/4e5c7a6b-3968-40d1-92f9-6ad0cfd306d6">